src/radio_instance.c

```diff
@@ -121,20 +127,32 @@ typedef struct radio_interface_desc {
 
 static const RadioInterfaceDesc radio_interfaces[] = {
     {
-        RADIO_INTERFACE_1_2,
-        RADIO_1_2,
+        RADIO_INTERFACE_1_4,
+        RADIO_1_4,
         radio_indication_ifaces + 0,
         radio_response_ifaces + 0,
+    },
+    {
+        RADIO_INTERFACE_1_3,
+        RADIO_1_3,
+        radio_indication_ifaces + 1,
+        radio_response_ifaces + 1,
+    },
+    {
+        RADIO_INTERFACE_1_2,
+        RADIO_1_2,
+        radio_indication_ifaces + 2,
+        radio_response_ifaces + 2,
     },{
         RADIO_INTERFACE_1_1,
         RADIO_1_1,
-        radio_indication_ifaces + 1,
-        radio_response_ifaces + 1
+        radio_indication_ifaces + 3,
+        radio_response_ifaces + 3
     },{
         RADIO_INTERFACE_1_0,
         RADIO_1_0,
-        radio_indication_ifaces + 2,
-        radio_response_ifaces + 2
+        radio_indication_ifaces + 4,
+        radio_response_ifaces + 4
     }
 };
```

Comment:
```c
G_STATIC_ASSERT(G_N_ELEMENTS(radio_interfaces) == RADIO_INTERFACE_COUNT);
```

Reply:
Why it wasn't there before? When those were only a few lines of code and they all fit nicely into a single, even small, screen, it wasn't that important. Now it's getting easier to miss something.

Reply:
Done.